src/services/axios.ts

```diff
@@ -1,12 +1,14 @@
-import axios, { AxiosInstance, AxiosResponse } from 'axios';
-
+import axios, { AxiosError, AxiosInstance, AxiosResponse } from 'axios';
+import { useAuthStore } from '@/stores/authStore';
 const instance: AxiosInstance = axios.create({
   baseURL: process.env.NEXT_PUBLIC_URL,
 });
 
```

Comment:
지금 보니까 여기 NEXT_PUBLIC_URL 없을 때 에러처리 하나 추가해주면 좋을 것 같네요. 
```suggestion
if (!process.env.NEXT_PUBLIC_URL) {
  throw new Error('NEXT_PUBLIC_URL is not defined');
}

```

Reply:
<img width="988" alt="image" src="https://github.com/user-attachments/assets/be4e691b-f15d-4dcc-8660-dae5c21f6532">

이부분은 넣으면 빌드 시 오류나서 제외하겟슴다